backport_diff_generator.py

```diff
@@ -101,12 +104,17 @@ def download_gerrit_revision(patch_set, revision_no) -> str or None:
         revision=revision_no, patch_set=patch_set))
 
 
+def validate_revision(rev):
```

Comment:
This might not be necessary if we default to PatchSet 1.

Reply:
What's your suggestion on this?